src/components/Error/NotFoundPage.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Row, Col, Button } from 'reactstrap';
+import { isExpectedPath } from 'kpmp-common-components';
```

Comment:
I think we can get rid of the isExpectedPath import (and thus the rest of kpmp-common-components)